src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -654,7 +734,7 @@ def resume(self) -> None:
                 kubernetes_context=kubernetes_context
             )
 
-        if not self.is_daemon_running:
+        if not self.skip_ui_daemon_provisioning and not self.is_daemon_running:
```

Comment:
Do we need this explicit check for `self.skip_ui_daemon_provisioning` here? I guess it's already covered in `self.is_daemon_running` right? (Same thing on the next change below)

Reply:
True, we don't need it here, but we do need it below, because the daemon is always reported as running if `self.skip_ui_daemon_provisioning`